pkg/planner/core/find_best_task.go

```diff
@@ -181,14 +181,18 @@ func rebuildChildTasks(p *logicalop.BaseLogicalPlan, childTasks *[]base.Task, pp
 	return nil
 }
 
+// @first: indicates the best task returned.
```

Comment:
How about also adding variable names to all returned values like `(bestTask base.Task, planCnt int64, hintApplied bool, err error)`?